vcenter/vcenter.py

```diff
@@ -449,6 +442,120 @@ def stop(self, uuid):
                 settings.raven.captureException(exc_info=True)
                 self.__sleep_between_tries()
 
+    def _take_screenshot_to_datastore(self, uuid):
+        """
+        Takes screenshot of VM and saves it in datastore
+        :param uuid: machine uuid
+        :return: tuple; name of datastore (where screenshot is saved) and path to screenshot in datastore
+        """
+        self.__logger.debug('-> take_screenshot()')
+        self.__check_connection()
+        vm = self.content.searchIndex.FindByUuid(None, uuid, True)
+        if vm is None:
+            raise Exception(f'machine {uuid} not found')
```

Comment:
My  concern is also this one, in other methods sometimes the exception is handled, sometimes not. (e.g. `start()` throws an exception, `stop()` does not. I'd vote for unify the approach in this api class)

Reply:
yes it should be unified but in different merge request; this is protected method not mentioned to be called from outside

Reply:
So unify in a way that we won't throw exceptions like here?